src/rpc/util.cpp

```diff
@@ -272,11 +272,12 @@ UniValue DescribeAddress(const CTxDestination& dest)
 
 unsigned int ParseConfirmTarget(const UniValue& value, unsigned int max_target)
 {
-    int target = value.get_int();
-    if (target < 1 || (unsigned int)target > max_target) {
-        throw JSONRPCError(RPC_INVALID_PARAMETER, strprintf("Invalid conf_target, must be between %u - %u", 1, max_target));
+    const int target{value.get_int()};
```

Comment:
What is the advantage of this construction over the previous line?

Reply:
Little: constness, and while changing the line, updated the style. It's not essential, to be sure. Initially, the function was touched to fix the error message. Then, I saw the function has two old C-style conversions and replaced them with one preferred static cast. Then, made the non-mutated variables const.